_posts/2020-03-04-developer-survey-results.md

```diff
@@ -0,0 +1,12 @@
+---
+category: news
+by: Julien Richard-Foy, Scala Center
+title: "Results of the 2019 Scala Developer Survey"
+---
+
+You have been about 4 thousands Scala developers to fill the survey and we are grateful
+to you! The survey results help us identify the trends in how you use the language in
+your day-to-day workflow, and what could be improved in the tooling or ecosystem.
```

Comment:
This is more of a nitpick, but after a comma, you must begin a new independent clause (with its own verb). It's fixable by omitting the comma, but then the sentence as a whole becomes somewhat hard to parse. It's a bit hard even with the comma. Perhaps aid the reader with bullets:

> The survey results help us identify:
> * trends in how you use the language in your daily work
> * what could be improved in tooling or the ecosystem

(note also the slight wording adjustments)